python/lsst/qserv/tests/benchmark.py

```diff
@@ -145,6 +145,9 @@ def runQueries(self, stopAt=MAX_QUERY):
                                       str(pragmas))
                     column_names = 'noheader' not in pragmas
                     sqlInterface.execute(qText, outFile, column_names)
+                    if 'sortresult' in pragmas:
+                        sortcmd = 'sort %s -o %s' % (outFile, outFile)
+                        os.system(sortcmd)
```

Comment:
Queries that were previously failing because results were coming in random order are now passing in qserv integration test, and they are now enabled. So this is the ultimate test / proof for us.